src/rpc/core_rpc_server.cpp

```diff
@@ -65,6 +65,12 @@ namespace
       reasons += ", ";
     reasons += reason;
   }
+
+  // assumes power of 2 quantum
+  uint64_t round_up(uint64_t value, uint64_t quantum)
+  {
+    return (value + quantum - 1) & ~(quantum - 1);
```

Comment:
Won't work. Should be
```suggestion
    return (value + quantum - 1) / quantum * quantum;
```

Reply:
Why would it not work ? I'll change to that anyway.

Reply:
due to the const we are using, it doesn't meet `power of 2` requirement